### ISSUE TITLE
Simple message system

### ISSUE THREAD

**Issue by ilyabyar** [MikeAndrianov/easy_learning#27]
I think that our app is need in a messaging system. I think that it should be simple, like in [onliner.by](http://onliner.by). Are there any suggestions?

**Comment by MikeAndrianov** [comment]
Yep.  It's a good idea. I suppose we should create something simple as you said. 
For a start add ability to send/receive messages directly one to one (but use **has_many through** association).
Your should simply add the model _Message_, link it with _User_, create controller (index, show, delete) and views. Also it would be great if you will add indicator of **unread messages** (something like on the facebook).
